Wishlist/wishlist.css

```diff
@@ -0,0 +1,31 @@
+@import url("/style.css");
+.card-container {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
```

Comment:
for gap here you use px unit and next you are using rem, be consisistent while using units